src/main.ts

```diff
@@ -3,67 +3,83 @@ import { LightSheetOptions } from "./main.types.ts";
 import Sheet from "./core/structure/sheet.ts";
 import { CellInfo } from "./core/structure/sheet.types.ts";
 import Events from "./core/event/events.ts";
-
+import LightSheetHelper from "../utils/helpers.ts";
+import { DefaultRowCount, DefaultColCount } from "../utils/constants.ts";
 export default class LightSheet {
-  ui: UI;
+  #ui: UI;
   options: LightSheetOptions;
   sheet: Sheet;
   events: Events;
+  onCellChange?;
```

Comment:
Do we need this still? I removed it in #85 since cell change can now be reacted to using `Events::on`.